### PR TITLE
feat(admin): admin area improvements

### LINKED ISSUE #10325: Admin is a dreary, sad place
Copy CSS from Aalborg, add some color. We spend a lot of time there. We should decorate.

### LINKED ISSUE #11028: master admin: "Logged in as ..." looks like a link
Due to output/url change I assume.

### PR DESCRIPTION
**Pull request by mrclay**
- More subtle colors (GitHub)
- Large headings not bold
- Cleaner admin menu
- Logged in user icon
- Fixed hover menu styles
- "Dashboard" is at `/admin`
- "Add widgets" has gear icon
- Highlight site name in header

Fixes #10325
Fixes #11028

admin

### CODE REVIEW

**Comment by jdalsem** [comment]
gear icon for 'add widgets' is a bit strange... why not a plus? or circle-plus?

**Comment by jdalsem** [comment]
Merging it as the rest looks fine by me

File: views/default/page/layouts/widgets/add_button.php [hunk 1]
```diff
@@ -17,8 +17,8 @@
 		[
 			'name' => 'widgets_add',
 			'href' => 'javascript: void(0)',
-			'text' => elgg_echo('widgets:add'),
-			'link_class' => 'elgg-button elgg-button-action elgg-lightbox',
+			'text' => elgg_view_icon('cog') . ' ' . elgg_echo('widgets:add'),
```
Comment (jdalsem) [inline views/default/page/layouts/widgets/add_button.php]: next time use the icon parameter to add icons to menu items

Comment (mrclay) [inline views/default/page/layouts/widgets/add_button.php]: I couldn't find docs for it. Where are they?

Comment (jdalsem) [inline views/default/page/layouts/widgets/add_button.php]: as menu items use output/url you can find docs in that view about 'icon' and 'badge'

**Pull request merged**